### PR TITLE
Create it.po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+msgid "Any files"
+msgstr "Qualsiasi file"
+
+#: src/window.py:235
+msgid "Save changes to this document?"
+msgstr "Vuoi salvare le modifiche a questo documento?"
+
+#: src/window.py:236
+msgid "If you don't, changes will be lost."
+msgstr "In caso contrario, le modifiche andranno perse"
```
Comment (ShikiOkasaka) [inline po/it.po]: Shouldn’t there be a period at the end?

File: po/it.po [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+msgid "Close _Without Saving"
+msgstr "Chiudi _Senza Salvare"
+
+#: src/window.py:270
+msgid "Font"
+msgstr "Font"
+
+#: src/window.py:271
+msgid "The quick brown fox jumps over the lazy dog."
+msgstr "La veloce volpe marrone salta sopra il cane pigro."
```
Comment (ShikiOkasaka) [inline po/it.po]: We don’t need to use a direct translation here. What’s the popular pangram in Italy these days?
See https://it.wikipedia.org/wiki/The_quick_brown_fox_jumps_over_the_lazy_dog

Comment (albanobattistella) [reply po/it.po]: > We don’t need to use a direct translation here. What’s the popular pangram in Italy these days? See https://it.wikipedia.org/wiki/The_quick_brown_fox_jumps_over_the_lazy_dog

HI,
there is no specific one but one as written is:"Pranzo d'acqua fa volti sghembi" or "Ma la volpe col suo balzo ha raggiunto il quieto Fido"

Comment (ShikiOkasaka) [reply po/it.po]: Will you please pick one that is often used in the font selection dialog in Italy?

**Comment by ShikiOkasaka** [comment]
LGTM

**Pull request merged**